src/js/components/AppListComponent.jsx

```diff
@@ -80,8 +76,8 @@ function updateGroupNode(group, app) {
   group.tasksRunning += app.tasksRunning;
   group.totalCpus += app.totalCpus;
   group.totalMem += app.totalMem;
-  group.health = getGroupHealth(group.health, app);
-  group.status = getGroupStatus(group.status, app);
+  group.status = getGroupStatus(group.status, app.status);
```

Comment:
Please keep it alphabetically.
I know, we have a lot of places where we were careless.. :)